tsconfig.json

```diff
@@ -1,48 +1,44 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "ES2015",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "target": "ES2015",                       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "module": "commonjs",                          /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
     "lib": ["es2017"],                             /* Specify library files to be included in the compilation. */
```

Comment:
I think our guidelines say not to include anything in `lib` because it will require downstream users to also include it. The eslint plugin at least checks that it is present and set to `[]`

Reply:
Hmm let me try it!